### ISSUE TITLE
openbh 5.0.011 (2022-03-08) looks for `/usr/bin/bcm3148 ` (trailing space) instead of `/usr/bin/bcm3148`

### ISSUE THREAD

**Issue by pcfe** [BlackHole/enigma2#99]
### Description

openbh 5.0.011 (2022-03-08) looks for `/usr/bin/bcm3148 ` (trailing space) instead of `/usr/bin/bcm3148`

### Steps to reproduce

1. Install _OpenBh 5.0_, specifically 2022-03-08) on a _Vu+ Duo 4K SE BT_ equipped with one DVB-C tuner card and one DVB-T2 tuner card. Since I have no plans to use the hardware's Bluetooh functionality, I picked `openbh-5.0.011.release-vuduo4kse-1205265b_usb.zip` without worrying whether I need `bt` in the image name or not.
2. Attempt to _Service scan_ the DVB-C tuner (currently there is no antenna connected to the DVB-T2 tuner)

### Actual result

E2 shows the following error pop up

> Cable scan executable utility not found '/usr/bin/bcm3148 '!

![Vu+ Duo4K SE fails to autoscan](https://user-images.githubusercontent.com/3717066/158070415-875f94a6-82be-4691-b177-48933fe881f3.jpg)

### Expected result

E2 calls the scan binary without a trailing space

#### Workaround

Primitively worked around by copying `/usr/bin/bcm3148` to one with a trailing space (`/usr/bin/bcm3148 `)

```bash
root@vuduo4kse:~# cp /usr/bin/bcm3148 /usr/bin/bcm3148\ 
root@vuduo4kse:~# ls -l /usr/bin/bcm3148*
-rwxr-xr-x    1 root     root         12426 Feb 13 22:40 /usr/bin/bcm3148
-rwxr-xr-x    1 root     root         12426 Mar 13 17:29 /usr/bin/bcm3148 
root@vuduo4kse:~# 
```

After doing that, I can trigger a _Service scan_ just fine in E2.

#### Tuner details

The vendor I bought the box (with pre-installed tuners) from calls it
_VU+ Duo 4K SE BT FBC Twin PVR 1xC/T2 bk 1xDVB-C/1xDVB-T2 FBC Twin Tuner PVR BT_

These are the tuner details as seen in OpenWebif

```text
Tuner A | Vuplus DVB-C NIM(BCM3158) (DVB-C)
Tuner B | Vuplus DVB-C NIM(BCM3158) (DVB-C)
Tuner C | Vuplus DVB-C NIM(BCM3158) (DVB-C)
Tuner D | Vuplus DVB-C NIM(BCM3158) (DVB-C)
Tuner E | Vuplus DVB-C NIM(BCM3158) (DVB-C)
Tuner F | Vuplus DVB-C NIM(BCM3158) (DVB-C)
Tuner G | Vuplus DVB-C NIM(BCM3158) (DVB-C)
Tuner H | Vuplus DVB-C NIM(BCM3158) (DVB-C)
Tuner I | Vuplus DVB-T NIM(BCM3466) (MultiType DVB-C/T2)
Tuner J | Vuplus DVB-T NIM(BCM3466) (MultiType DVB-C/T2)
```

**Comment by dpeddi** [comment]
Thank you for reporting and providing your workaround... It have been helpful for locating the issue:

The issue have been introduced in commit 94810b81e1cf59966164b0586a157f2c11f4d725 and we are working for a fix... stay tuned.

**Comment by dpeddi** [comment]
Please wait a couple of day for build to complete... ;-)

**Comment by pcfe** [comment]
to conclude this, I successfully verified the fix is present in 5.0.013 (build available since early this morning).
Thanks again for the quick fixing.

### verification details

#### remove existing workaround

I undid my workaround (`rm /usr/bin/bcm3148\ `)

#### apply update

I applied the update to 5.0.013 via _Menu / Setup / System / Software update / Online update / Upgrade and reboot system_.

#### verify version

Box now reports (as expected) in Web Interface

> Software
> --
> System OE: | OE-Alliance 5.0
> Firmware version: | openbh 5.0.013 (2022-03-18)
> Kernel / Drivers: | 4.1.45 / 20210527

#### scan DVB-C

As expected, E2 scans just fine with 5.0.013
![image](https://user-images.githubusercontent.com/3717066/159125198-3865c265-90db-4158-8bf9-dda03856c244.png)
